jobs/trucker.md

```diff
@@ -8,9 +8,16 @@ title: Trucker
 | Necesită vehicul personal? | Nu, jobul se practică cu un vehicul special. |
 | Necesită permis de conducere? | Nu, dar ajută (categoria C). |
 | Necesită iteme specifice? | Nu. |
-| Posibilitate găsire obiecte speciale? | Nu. |
+| Posibilitate găsire obiecte speciale? | [Da](#nou). |
 | Locație NPC Trucker | [Palmer-Taylor Power Station](https://i.imgur.com/tOFQmL1.png)  |
 
+## Nou
+
+**Jucatorii ce detin minim skill Silver pot opta pentru curse ilegale. Cursele ilegale ofera jucatorilor un boost de 10%. Exista o sansa de 5% de a primi wanted la efectuarea unei astfel de curse.**
+
+- Trucker:
+    - Explosive: sansa de drop de 20% la fiecare $300 castigati in urma efectuarii unei curse ilegale
+
```

Comment:
**Wanted-ul primit in urma curselor ilegale se va cumula ca si timp?**

Nu, wanted-ul pe care il primesti in timpul curselor ilegale din cadrul jobului "Trucker" nu se cumuleaza, doar se va reseta timpul daca vei primi din nou wanted la alta cursa ilegala.